README.md

```diff
@@ -1,78 +1,76 @@
-# Custom functions in Excel (Preview)
+# Custom functions in Excel (preview)
 
-Learn how to use custom functions in Excel (similar to user-defined functions, or UDFs). Custom functions are JavaScript functions that you can add to Excel, and then use them like any native Excel function (for example =Sum). This sample accompanies the [Custom Functions Overview](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview) topic.
+Custom functions enable you to add new functions to Excel by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as `SUM()`. 
+
+This repository contains a simple custom functions add-in project that you can use to learn about developing custom functions in Excel. You may also choose to use this project as a basis for creating your own custom functions project. For more detailed information about custom functions in Excel, see the [Custom functions overview](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-overview) article in the Office Add-ins documentation.
 
 ## Table of Contents
-* [Change History](#change-history)
+
+* [Change history](#change-history)
 * [Prerequisites](#prerequisites)
-* [To use the project](#to-use-the-project)
+* [Using this project](#using-this-project)
 * [Making changes](#making-changes)
 * [Debugging](#debugging)
-* [IntelliSense for the JSON file in Visual Studio Code](#intellisense-for-the-json-file-in-visual-studio-code)
 * [Questions and comments](#questions-and-comments)
 * [Additional resources](#additional-resources)
 
-## Change History
+## Change history
```

Comment:
Do we need the "Change history" section? Or perhaps put it at then end after "Additional resources"...

Reply:
@akrantz thanks for raising this question. IMO, an out-of-date **Change history** section is worse than having no **Change history** section at all.  (I almost removed it during my first pass of README updates.) Given that nobody's been maintaining this section, I'm going to remove it for now; someone can always add it back later if they feel strongly about updating/maintaining change history info going forward.